people.rkt

```diff
@@ -71,7 +71,7 @@
                   #:title "Professor"
                   #:website "http://ccs.neu.edu/home/will"
                   #:e-mail "will@ccs.neu.edu"
-                  #:history @list[@a[href: "http://www.cesura17.net/~will/Professional/"]{www.cesura17.net/~will/Professional/}]
+                  #:history @list[]
```

Comment:
I removed it because it doesn't actually provide any history - finding out more details on that is something that I'm working on now. It could stay in, though.